languages/clojure/exercises/concept/basic-functions/.doc/introduction.md

```diff
@@ -0,0 +1,45 @@
+In Clojure, functions are your basic tools for manipulating data.
+
+Functions can be defined by using the `def` and `fn` functions together, but in practice Clojure functions are almost always defined with the `defn` macro:
+
+```clojure
```

Comment:
Quoting the [documentation](https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md#step-1-add-docsintroductionmd):

> The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms that the student can Google if they so want, is preferable over including any code samples or an extensive description.

As such, could you perhaps remove the code examples in this file? The rest of the content is very well written and provides the student with enough context to attempt to solve the exercise.

Reply:
removed code examples. 👍